Boundaries.playground/Contents.swift

```diff
@@ -1,6 +1,15 @@
 import Boundaries
 import Optics
 import Prelude
+import Dispatch
+import PlaygroundSupport
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+enum TestEffect: EffectProtocol {
```

Comment:
Would be really cool to play around more with effect composition and bundling common Foundation-based effects (file system, network requests) with the lib. I think that'll inform iterations on this design.